composer.json

```diff
@@ -51,7 +52,7 @@
     "ext-pdo_sqlite": "For running functional tests out-of-the-box this is required"
   },
   "scripts": {
-    "post-create-project-cmd": "./flow welcome",
+    "post-create-project-cmd": "./flow setup:index",
```

Comment:
```suggestion
    "post-create-project-cmd": "./flow setup",
```

Reply:
So I guess to not confuse anyone the base distribution should also use `"post-create-project-cmd": "./flow setup",` instead of `"post-create-project-cmd": "./flow setup:index",`?

see: https://github.com/neos/neos-base-distribution/pull/61

Reply:
Yes

Reply:
And also target 8.0